features/step_definitions/work_steps.rb

```diff
@@ -564,6 +564,14 @@
   step "the periodic tag count task is run"
 end
 
+When "I browse works in {string}" do |language_name|
```

Comment:
How about
```suggestion
When "I browse works in language {string}" do |language_name|
```
to make it 100% clear that it's a language, and not say a fandom or other tag?